compiler/AST/flags.cpp

```diff
@@ -62,9 +64,9 @@ initFlags() {
 }
 
 void writeFlags(FILE* fp, Symbol* sym) {
-  form_Map(FlagMapElem, e, flagMap) {
-    if (sym->flags[e->value]) {
-      fprintf(fp, " \"%s\"", e->key);
+  for (int flagNum = FLAG_UNKNOWN+1; flagNum < NUM_FLAGS; flagNum++) {
```

Comment:
FLAG_UNKNOWN+1 seems a tiny bit crufty.  Maybe add a new enumerator FLAG_FIRST_FLAG or something like that?  Purely optional.